src/BenchmarkDotNet/Extensions/StatisticsExtensions.cs

```diff
@@ -36,16 +36,16 @@ public static string ToTimeStr(this Statistics s, TimeUnit unit = null, bool cal
             var ci = s.ConfidenceInterval;
             string ciMarginPercent = (ci.Margin / s.Mean * 100).ToStr("0.00");
             double mValue = MathHelper.CalculateMValue(s);
-            builder.AppendLine($"Mean = {s.Mean.ToTimeStr(unit)}, StdErr = {s.StandardError.ToTimeStr(unit)} ({errorPercent}%); N = {s.N}, StdDev = {s.StandardDeviation.ToTimeStr(unit)}");
-            builder.AppendLine($"Min = {s.Min.ToTimeStr(unit)}, Q1 = {s.Q1.ToTimeStr(unit)}, Median = {s.Median.ToTimeStr(unit)}, Q3 = {s.Q3.ToTimeStr(unit)}, Max = {s.Max.ToTimeStr(unit)}");
-            builder.AppendLine($"IQR = {s.InterquartileRange.ToTimeStr(unit)}, LowerFence = {s.LowerFence.ToTimeStr(unit)}, UpperFence = {s.UpperFence.ToTimeStr(unit)}");
-            builder.AppendLine($"ConfidenceInterval = {s.ConfidenceInterval.ToTimeStr(unit)}, Margin = {ci.Margin.ToTimeStr(unit)} ({ciMarginPercent}% of Mean)");
+            builder.AppendLine($"Mean = {s.Mean.ToTimeStr(unit, encoding: encoding)}, StdErr = {s.StandardError.ToTimeStr(unit, encoding: encoding)} ({errorPercent}%); N = {s.N}, StdDev = {s.StandardDeviation.ToTimeStr(unit, encoding: encoding)}");
+            builder.AppendLine($"Min = {s.Min.ToTimeStr(unit, encoding: encoding)}, Q1 = {s.Q1.ToTimeStr(unit, encoding: encoding)}, Median = {s.Median.ToTimeStr(unit, encoding: encoding)}, Q3 = {s.Q3.ToTimeStr(unit, encoding: encoding)}, Max = {s.Max.ToTimeStr(unit, encoding: encoding)}");
+            builder.AppendLine($"IQR = {s.InterquartileRange.ToTimeStr(unit, encoding: encoding)}, LowerFence = {s.LowerFence.ToTimeStr(unit, encoding: encoding)}, UpperFence = {s.UpperFence.ToTimeStr(unit, encoding: encoding)}");
+            builder.AppendLine($"ConfidenceInterval = {s.ConfidenceInterval.ToTimeStr(encoding, unit)}, Margin = {ci.Margin.ToTimeStr(unit, encoding: encoding)} ({ciMarginPercent}% of Mean)");
```

Comment:
We need another overload for `ToTimeStr` which allows writing `ToTimeStr(unit, encoding)` isntead of `ToTimeStr(unit, encoding: encoding)`